setup.py

```diff
@@ -43,6 +43,7 @@
         "MutatorMath>=2.0.2",
         "defcon>=0.2.1",
         "booleanOperations>=0.6.4",
+        "mock>=2.0.0",
```

Comment:
I like that you used the `mock` module to write the unit tests.

However, if anything,`mock` is only a _test_ requirement, not an _install_ requirement, so it should not be in `install_requires`.

Similarly, the `requirements.txt` file should not contain `mock` either, as that's used for specifying install requirements with pinned-down version; if we want the tests to be run with fixed versions of the test requirements, we could create a separate `test-requirements.txt` file or similar.